custom_components/solarman/inverter_definitions/deye_hybrid.yaml

```diff
@@ -874,5 +910,5 @@ parameters:
         rule: 1
         registers: [0x00F8]
         value:
-          on: 0x00FF
-          off: 0x0000
+          "on": 0x00FF
+          "off": 0x0000
```

Comment:
While this was working fine without escaping for this register (Time of Use) because it does seem support writing 1 instead of 255 to "switch on" this is not correct configuration due to this:

https://github.com/krasnoukhov/ha-solarman/blob/95880ac14b2e03bbc3fb21d369605af4d1131d34/custom_components/solarman/switch.py#L49-L54

In yaml `on` equals `true` so `self._value_on` was not set to 255 but rather 1

I can confirm it works with both 1 and 255 so this stuff could be removed potentially but I wanted to change this for consistency